src/get-started/test-service.md

```diff
@@ -248,7 +246,8 @@ executed.
 We should check that Alice did not send her tokens to nowhere:
 
 ```rust
-assert_eq!(alice_wallet.balance, 100);
+assert!(try_get_wallet(&testkit, &alice.public_key()).is_none());
```

Comment:
`try_get_wallet` and `get_wallet` functions need to be defined beforehand; currently, they appear out of nowhere.